b2share/config.py

```diff
@@ -190,3 +190,13 @@
         'serializer': 'b2share.modules.records.serializers.oaipmh_marc21_v1',
     },
 }
+
+
+CFG_HANDLE_SYSTEM_BASEURL = 'http://hdl.handle.net'
+CFG_FAIL_ON_MISSING_PID = False
+
+# The following config variables are used to create EPIC PIDs
+# CFG_EPIC_USERNAME = 0000
+# CFG_EPIC_PASSWORD = ''
```

Comment:
are this account and password still valid? if so they should be removed. But also, why do we need to add this commented code?